packages/hooks/src/usePollSubgraph.ts

```diff
@@ -0,0 +1,45 @@
+const usePollSubgraph = ({
```

Comment:
Is this hook really required? 

As you can see in other hooks such as `useInvoiceCreate`, I had replaced the usage of this hook with `await waitForSubgraphSync(publicClient.chain.id, txData.blockNumber);`

This is simpler and cleaner and achieves the same result. I'd rather not add this hook back in here. If required you may have the hook in the DM codebase. Lets not have it in the SI codebase @sshmm 🙏🏽